packages/block-editor/src/hooks/layout.scss

```diff
@@ -1,23 +1,13 @@
-.block-editor-hooks__layout-controls-units {
-	display: flex;
-	flex-direction: column;
-	gap: $grid-unit-20;
-}
-
-.block-editor-block-inspector .block-editor-hooks__layout-controls-unit-input {
-	margin-bottom: 0;
-}
-
-.block-editor-hooks__layout-controls-reset {
```

Comment:
Is there any chance any of these removed classes were used for additional styling?

Reply:
Nope, we're good 👍